app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/ModalForm.tsx

```diff
@@ -172,12 +271,12 @@ export const ModalForm = ({
               </div>
             </InfoDetails>
           </div>
-        </div>
+        </section>
       )}
       {item.type === FormElementTypes.textField ||
-        (FormElementTypes.textArea &&
+        (item.type === FormElementTypes.textArea &&
```

Comment:
Unrelated bug fix- this conditional was incorrect

Reply:
I don't think anything is going to render here for textField --- I think we need a further adjustment.